src/main/java/cam72cam/immersiverailroading/Config.java

```diff
@@ -19,6 +19,9 @@
 		@Comment({ "Enable Boiler Explosions" })
 		public static boolean explosionsEnabled = true;
 
+		@Comment({ "Enable environmental damage of Boiler Explosions"})
+		public static boolean explosionEnvDamageEnabled = true;
+		
```

Comment:
Why is this in here?

Reply:
idk, this is not new. Had some Problems with Github Desktop recently